tests/test_continue_simulation.py

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright (C) 2018-2019 University of Oxford
+#
+# This file is part of msprime.
+#
+# msprime is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# msprime is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with msprime.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Test cases for continue_simulation.
+"""
+import pytest
+
+import msprime
+
+
+class TestContinueSimulation:
```

Comment:
I think we should write a verification function, that would
- check that subsetting to all nodes present before `time` gets back the original tables
- check that all roots of all trees are older than `time`

Reply:
On that first condition, which properties could be appropriate and sufficient to do the comparison on? We can get back aspects of the original tables, but with altered flags and times in the nodes and mutations tables. An edge table comparison could be fairly comprehensive and feasible.

Reply:
Good question, and this can be pretty tricky. ButI think if you re-mark the nodes present at `time` as samples, and then simplify to them, you should get back exactly the original tables?  Might anything else change?

Reply:
Oh - sites and mutations shouldn't have changed.